src/subscription_manager/i18n.py

```diff
@@ -51,10 +54,100 @@ def configure_gettext():
     gettext.bind_textdomain_codeset(APP, 'UTF-8')
     locale.bind_textdomain_codeset(APP, 'UTF-8')
 
+
 translation = gettext.translation(APP, fallback=True)
 if six.PY3:  # gettext returns unicode in Python 3
     ugettext = translation.gettext
     ungettext = translation.ngettext
 else:
     ugettext = translation.ugettext
     ungettext = translation.ungettext
+
+
+class Locale(object):
+    """
+    Class used for changing languages on the fly
+    """
+
+    languages = {}
```

Comment:
I think this should be called translations, since it is a dict with `gettext.translation` values.